session/handler.go

```diff
@@ -263,6 +269,115 @@ func (h *Handler) adminDeleteIdentitySessions(w http.ResponseWriter, r *http.Req
 	w.WriteHeader(http.StatusNoContent)
 }
 
+// Session List Request
+//
+// The request object for listing sessions in an administrative context.
+//
+// swagger:parameters adminListSessions
+// nolint:deadcode,unused
+type adminListSessionsRequest struct {
+	// Items per Page
+	//
+	// This is the number of items per page to return.
+	// For details on pagination please head over to the [pagination documentation](https://www.ory.sh/docs/ecosystem/api-design#pagination).
+	//
+	// required: false
+	// in: query
+	// default: 250
+	// min: 1
+	// max: 1000
+	PageSize int `json:"page_size"`
```

Comment:
Use structs from ory/x here?

Reply:
yep. [will basically use this](https://github.com/ory/x/pull/619)

Reply:
Fixed to use `keysetpagination.RequestParameters`